### PR TITLE
Remove coveralls

### PR DESCRIPTION
**Pull request by deivid-rodriguez**
* First commit addresses https://github.com/jujugrrr/rubocop/commit/24c04b34ad97478482040bc993916a36564adf4e.
* Second commit helps preventing things like https://github.com/jujugrrr/rubocop/commit/24c04b34ad97478482040bc993916a36564adf4e in the future.
* Third commit address https://github.com/bbatsov/rubocop/pull/2408#issuecomment-155560125

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -4,8 +4,16 @@ source 'https://rubygems.org'
 
 gemspec
 
+gem 'rake', '~> 10.1'
+gem 'rspec', '~> 3.3.0'
+gem 'yard', '~> 0.8'
+gem 'simplecov', '~> 0.10'
+# Pinning tins due to 1.7.0 incompatibility with Ruby < 2.0
+# https://github.com/flori/tins/blob/v1.7.0/tins.gemspec#L19
+gem 'tins', '<= 1.6.0'
```
Comment (bbatsov) [inline Gemfile]: So, that's a dependency of simplecov after all?

**Comment by bbatsov** [comment]
So, I guess I have to find out how to sign up for this service. :-)

File: Gemfile (hunk 1 above)
Comment (deivid-rodriguez) [inline Gemfile]: It's a dependency of coveralls.

**Comment by deivid-rodriguez** [comment]
Go to https://codeclimate.com/github/bbatsov/rubocop, and there's should be a coverage section (gray now) where you should be able to get a token.

**Comment by bbatsov** [comment]
The token is `a11b66bfbb1acdf220d5cb317b2e945a986fd85adebe29a76d411ad6d74ec31f`.

**Comment by deivid-rodriguez** [comment]
:green_heart: :green_heart:

**Comment by jujugrrr** [comment]
hehe cheers and sorry for the mistake :wink:

**Pull request merged**